src/containers/BookList.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Book from '../components/Book';
+import {connect} from 'react-redux';
+import {PropTypes} from 'prop-types';
+import { removeBook, changeFilter }from '../actions/index';
+import { CategoryFilter } from './CategoryFilter';
+
+function BookList(props) {
+
+    const booksList = () => {
+        if (props.filter === 'All'){
+            return props.books;
+        } else {
+            return props.books.filter(book => book.category === props.filter);
+        }
```

Comment:
at all times a filter is in act, `All` or a specified one. it would be nice to do this at the state level and the props should be what has been filtered so we get just what we need and don't do this in the component.

Reply:
Can you elaborate more on this one ?

Reply:
@azdrenymeri  slack please

Reply:
Okay thanks .
Done